homeassistant/helpers/state.py

```diff
@@ -202,11 +203,12 @@ def state_as_number(state):
 
     Raises ValueError if this is not possible.
     """
-    if state.state in (STATE_ON, STATE_LOCKED, STATE_ABOVE_HORIZON,
-                       STATE_OPEN):
+    if state.state in (STATE_ON, STATE_LOCKED, STATE_ABOVE_HORIZON, STATE_OPEN,
+                       STATE_HOME, STATE_PLAYING):
```

Comment:
I personally feel that, in the context of "assigning" a value to a state, `STATE_HOME` and `STATE_PLAYING` naturally fall under a "1". If anything, we should be adding more states and mapping them to `1` and `0`.

However, I understand the schema issues this may bring. In fact, it's probably an issue for other components as well ([logentries for example](http://github.com/nunofgs/home-assistant/blob/c175f537efa0cad1640c47ba33e8f0dee3eef7ff/homeassistant/components/logentries.py#L42)). By changing the converted _state_ this PR is introducing a BC and possibly breaking alerts/graphs that people are using.

I'll revert this change but I really think an improved way of converting to a number should be developed, as Datadog (and possibly others) only handle numbers (not strings).